flink-table/flink-table-planner-blink/src/test/java/org/apache/flink/table/planner/operations/SqlToOperationConverterTest.java

```diff
@@ -1153,6 +1155,45 @@ public void testAlterTableDropConstraint() throws Exception {
 		parse("alter table tb1 drop constraint ct2", SqlDialect.DEFAULT);
 	}
 
+	@Test
+	public void testCreateViewWithMatchRecognize() {
+		Map<String, String> prop = new HashMap<>();
+		prop.put("connector", "datagen");
```

Comment:
nit: use `values` instead of `datagen`, `datagen` is not intended for testing as far as I know